docs/whatsnew/1.7.0/1_7_0.rst

```diff
@@ -0,0 +1,53 @@
+Pylake 1.7.0
+============
+
+.. only:: html
+
+Here is an overview of features that were released in Pylake `v1.7.0`.
+
+Calculating effective on-rates from a kymograph
+-----------------------------------------------
+
+We’ve introduced a new example analysis workflow for calculating the effective on-rate, :math:`k'_{on}` (the inverse of the binding time shown in Figure 1), for proteins binding to a target site.
```

Comment:
…(the inverse of the time *between* binding events as shown in Figure 1)

Reply:
Good catch. Fixing!